types/react-navigation/index.d.ts

```diff
@@ -268,6 +268,7 @@ export interface NavigationInitAction {
 export interface NavigationResetAction {
   type?: 'Navigation/RESET',
   index: number,
+  key?: string | null,
```

Comment:
![image](https://user-images.githubusercontent.com/5743863/27166550-2af56d6e-51ce-11e7-9a9d-80b502406b98.png)

It' s okay.